plugins/@grouparoo/sqlite/src/lib/test.ts

```diff
@@ -0,0 +1,20 @@
+import { TestPluginMethod } from "@grouparoo/core";
+
+export const test: TestPluginMethod = async ({ connection }) => {
+  const genericError = {
+    success: false,
+    message: "Could not connect to database.",
+  };
+
+  // Return if we don't have a connection object.
+  if (!connection) return genericError;
+
+  const query = `SELECT count(*) FROM sqlite_master WHERE type = 'table' AND name != 'android_metadata' AND name != 'sqlite_sequence';`;
```

Comment:
`android_metadata`?  Are there other strange os-specific tables we should be ignoring?

Reply:
I couldn't find a good list of this. But upon further inspection, anything that begins with `sqlite_` is reserved for internal use. Therefore, I've made the change to only filter out those tables beginning with `sqlite_` since any other table could be used for meta purposes, theoretically.